storage/test/test_akamai.py

```diff
@@ -46,7 +51,7 @@ def test_direct_download_cdn_specific(ipranges_populated, test_ip_range_cache, a
             "some/path",
             _TEST_BUCKET,
             _TEST_REGION,
-            None,
+            amz_encryption_token=token,
```

Comment:
The encryption token is not for Amazon. can you call it something else like `akamai_sig_encryption_key`